packages/webapp/src/components/Forms/SimpleCustomType/index.jsx

```diff
@@ -66,17 +68,25 @@ const PureSimpleCustomType = ({
   inputMaxChars = 100,
   validateInput,
   useHookFormPersist = () => ({}),
+  customRadioGroup,
```

Comment:
Can we make this even more generic-er? Like customizedFormInputs or customFormChildren or something? Radio group seems somewhat specific.

Reply:
Yeah that would be much better!